libraries/bsg_manycore_config.cpp

```diff
@@ -171,5 +171,8 @@ int hb_mc_config_init(const hb_mc_config_raw_t raw[HB_MC_CONFIG_MAX],
         config->io_host_credits_cap = idx;
 
 
+	config->dram_channels = raw[HB_MC_CONFIG_DRAM_CHANNELS];
+	config->dram_bank_size_words = raw[HB_MC_CONFIG_DRAM_BANK_SIZE_WORDS];
+
```

Comment:
Like I said to Leonard, it would be good to check these values for sanity

Reply:
I guess a sanity check is reasonable - like make sure dram channels is a power of 2 and its not ridiculous? Same maybe with bank_size_words?